x/strings/strings.go

```diff
@@ -1,11 +1,11 @@
 package strings
 
 // ArrayFn is the type signature of a string slice clearing fn.
-type ArrayFn func(values []string)
+type ArrayFn func(values [][]byte)
```

Comment:
Rename this file?

Reply:
I'm going to rename the file to `bytes.go` but not the package since the naming follows the assumption we are making across the board that `strings` are `[]byte`s.